README.md

```diff
@@ -139,7 +240,9 @@ rule:
     namespace: data-manipulation/checksum/crc32
     authors:
       - moritz.raabe@mandiant.com
-    scope: function
+    scopes:
```

Comment:
I think this rule is outdated and we can find a better example to show a basic rule.

Reply:
`create TCP socket` may be an intuitive one